django-stubs/db/models/query_utils.pyi

```diff
@@ -40,6 +40,8 @@ class Q(tree.Node):
         summarize: bool = ...,
         for_save: bool = ...,
     ) -> WhereNode: ...
+    def flatten(self) -> Iterator[Any]: ...
```

Comment:
```suggestion
    def flatten(self) -> Iterator[Node | NothingNode | ???]: ...
```
Not sure if anything else could be yielded

Reply:
Our typing of `Node.children` and `Q.__init__(args, kwargs)` typing is incomplete also, and looks wrong when I look at `flatten()` code.

If you're not interested in un-entangling those, I think best to use `Incomplete` with `from _typeshed import Incomplete`